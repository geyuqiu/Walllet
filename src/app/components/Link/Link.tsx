import React from "react";

type LinkProps = {
	href: string;
	className: string;
	children: React.ReactNode
};

export const Link = ({ href, className, children }: LinkProps) => (
		<a href={href}
		   className={className} >
			{children}
		</a>
	);
