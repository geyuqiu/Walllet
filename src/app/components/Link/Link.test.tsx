import {render} from "@testing-library/react";
import React from "react";

import {Link} from "./Link";

describe("Link", () => {
	it("should render", () => {
		const { container } = render(
			<Link href="https://test.de"
			      className="text-green-dark hover:text-green-darkest active:text-green-light font-semibold hover:underline">link
			</Link>
		);

		expect(container).toMatchSnapshot();
	});
});
