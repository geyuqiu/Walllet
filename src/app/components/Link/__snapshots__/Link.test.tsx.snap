// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link should render 1`] = `
<div>
  <a
    class="text-green-dark hover:text-green-darkest active:text-green-light font-semibold hover:underline"
    href="https://test.de"
  >
    link
  </a>
</div>
`;
